### PR TITLE
Add view About Self Care and Social Prescribing

### PR DESCRIPTION
**Pull request by SevaDer14**
PT: https://www.pivotaltracker.com/story/show/178911770
### User Story
```
As a Charity
To inform visitor about benefits of Self Care
We need a section telling about it
```
### Changes Proposed:
- create `AboutSelfCare` view
- adds `/about/self_care` route
- maps necessary Sections onto this view
### Picture
https://user-images.githubusercontent.com/79849155/127158270-76f3132d-81cf-4e7b-bbea-f4a243cf016a.mp4

### CODE REVIEW

File: src/views/AboutSelfCareView.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import SectionSelector from "../components/Section/SectionSelector";
+
+const AboutView = () => {
+  const [sections, setSections] = useState([]);
+
+  useEffect(() => {
+    const fetchPageData = async () => {
+      let response = await Sections.read("about_self_care");
+      setSections(response);
+    };
```
Comment (tochman) [inline src/views/AboutSelfCareView.jsx:14]: Does this pass the browser without warnings?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Client Application for Community Health West London
 
 `/`: Landing page (`<IndexView.jsx />`)
 
-`/about`: About section? (`<AboutView.jsx />`)
+`/about/us`: About section? (`<AboutView.jsx />`)
```
Comment (tochman) [inline README.md:11]: I assume that we need to update this section with more routes?

File: src/views/AboutSelfCareView.jsx (hunk 1 above)
Comment (SevaDer14) [reply src/views/AboutSelfCareView.jsx:14]: yes, no warning in console

File: README.md (hunk 2 above)
Comment (SevaDer14) [reply README.md]: fixed

**Pull request merged**